server/parsec/api.py

```diff
@@ -36,7 +36,7 @@
 }
 
 
-def api(fn: ApiFnWithSelf) -> ApiFnWithSelf:
+def api(fn: ApiFnWithSelf) -> ApiFnWithSelf:  # pyright: ignore[reportMissingTypeArgument] see TODO for Req/Rep above
```

Comment:
I wonder how hard it would be to fix that

Reply:
:shrug: That for another PR